cmake/Codegen.cmake

```diff
@@ -251,3 +251,9 @@ function(append_filelist name outputvar)
   list(APPEND ${outputvar} ${_tempvar})
   set(${outputvar} "${${outputvar}}" PARENT_SCOPE)
 endfunction()
+
+
+
+set(CPU_CAPABILITY_FLAGS ${CPU_CAPABILITY_FLAGS} PARENT_SCOPE)
```

Comment:
@quickwritereader 
Got a question when doing the final went through. Do we really need the following 3 lines?
line 1 and line 3 are exactly the same.
CPU_CAPABILITY_FLAGS and NUM_CPU_CAPABILITY_NAMES only appeared in this file.

Reply:
maybe while merging I messed this

Reply:
I'll remove